src/BootstrapAdminUi/templates/shared/helper/pagination.html.twig

```diff
@@ -25,5 +25,9 @@
                 {% endfor %}
             </div>
         </div>
+    {% else %}
+        <button type="button" class="btn btn-ghost-secondary dropdown-toggle" disabled>
+            {{ 'sylius.ui.show'|trans }} {{ paginator.maxPerPage }}
+        </button>
```

Comment:
To be able to reuse this elsewhere can we imagine extracting this to a proper template ?